### ISSUE TITLE
Automatic story generation

### ISSUE THREAD

**Issue by ennioVisco** [histoire-dev/histoire#198]
### Clear and concise description of the problem

First of all, thumbs up for this project, I stumbled on it a few hours ago, and I'm totally in love with the concept, and with what you are doing.

I want to challenge the developers with one idea: Histoire could be directly incorporated into the development flow, where  stories for basic components could be generated automatically by exploiting vue's SFC syntax.

### Suggested solution

Example of a well-documented component for a button:
```vue
<!-- components/buttons/MyButton.vue -->
<script>
// Javascript from https://github.com/vue-styleguidist/vue-styleguidist/tree/dev/packages/vue-docgen-api
export default {
  props: {
    /**
     * Color of the button.
     */
    color: {
      type: String,
      default: '#FCC'
    },
    /**
     * initial value to be passed but undocumented
     * @ignore
     */
    initialvalue: {
      type: Number,
      default: 0
    },
    /**
     * The size of the button allows only some values
     * @values small, medium, large
     */
    size: {
      default: 'normal'
    }
  }
}
</script>

<template>
    <button :width="size" ...>
        <slot /> 
    </button>
</template>

<docs lang="md">
# My button documentation
...
</docs>
```

It looks to me that already many information for a story are present, where the title of the story, the path etc. can call be derived from the present information. What do you think?

### Validations

- [X] Follow our [Code of Conduct](https://github.com/histoire-dev/histoire/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/histoire-dev/histoire/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://histoire.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/histoire-dev/histoire/issues) that requests the same feature to avoid creating a duplicate.

**Comment by a474516631** [comment]
Yep, I think that's a great idea too. In the process of using histoire, I also wrote a similar cli to do similar things. I think it greatly improves the efficiency of my writing documents

**Comment by Akryum** [comment]
related to #126 #41
